packages/desktop/webpack.config.js

```diff
@@ -12,6 +12,7 @@ const prod = mode === 'production'
 const hardcodeNodeEnv = typeof process.env.HARDCODE_NODE_ENV !== 'undefined'
 const SENTRY = process.env.SENTRY === 'true'
 const stage = process.env.STAGE || 'alpha'
+const os = process.env.OS || 'ubuntu-20.04'
```

Comment:
Is this necessary? Would `process.platform` not work?

Reply:
nice catch